locale/Makefile

```diff
@@ -1,6 +1,62 @@
 #
-# Makefile for PO merging and MO generation. For more info see
-# locale/README in the Foreman Core.
+# Makefile for PO merging and MO generation. More info in the README.
 #
-include ../.foreman_app/locale/Makefile
-DOMAIN = docker
+# make all-mo (default) - generate MO files
+# make check - check translations using translate-tool
+# make tx-update - download and merge translations from Transifex
+# make clean - clean everything
+#
+DOMAIN = foreman-docker
```

Comment:
underscore, to match https://github.com/theforeman/foreman-docker/blob/master/lib/foreman_docker/engine.rb#L36